webserver/script/validators.py

```diff
@@ -17,3 +17,8 @@ def validate_month(month):
 def validate_positive(value):
     if value < 0:
         raise ValidationError("Value must be a positive.")
+
+
+def validate_zipcode(zipcode):
+    if len(zipcode) != 5 or zipcode.isdigit():
```

Comment:
I just found I missed a `not` here. I just want to avoid typo from users' inputs, such as `9408g`.
And the reason why I chose to use `CharField` rather than `IntegerField` is there could be some zipcodes beginning with `0`. (I'm not sure if US has such kind of zipcodes...)